org.eclipse.xtext.ui.shared/src/org/eclipse/xtext/ui/shared/internal/Activator.java

```diff
@@ -76,21 +76,25 @@ protected void initializeInjector(BundleContext context) throws CoreException {
 		Module module = new SharedModule(context);
 		if (extensions.length != 0) {
 			int numberOfMixedInModules = 0;
+			StringBuilder sb = new StringBuilder();
+			sb.append("xtext.ui.shared: Multiple contributions to extension point 'org.eclipse.xtext.ui.shared.overridingGuiceModule'."
```

Comment:
Fine by me. 
A few nits:
- The sb can be initialized with that string literal directly.
- Having a prefix "xtext.ui.shared" is usually not necessary. This will be implicit from the loggers name.
- Please don't hardcode the extension point name but use the same logic that does the lookup or even use `point.getUniqueIdentifier()`